### PR TITLE
Fix gpio controller

### PR DESCRIPTION
**Pull request by MGBla**
- Add pin labels (#98)
- Teachpanel and Services can no interchangeably adjust the GPIOs (#87)

### CODE REVIEW

File: ur_robot_driver/src/hardware_interface.cpp [hunk 1]
```diff
@@ -533,18 +533,18 @@ void URPositionHardwareInterface::handleRobotProgramState(bool program_running)
 void URPositionHardwareInterface::initAsyncIO()
 {
   for (size_t i = 0; i < 18; ++i) {
-    standard_dig_out_bits_cmd_[i] = static_cast<double>(actual_dig_out_bits_[i]);
+    standard_dig_out_bits_cmd_[i] = -1;
```
Comment (urrsk) [inline ur_robot_driver/src/hardware_interface.cpp:536]: Please define ```-1``` as a constant with a meaningful name and use that instead

File: ur_robot_driver/src/hardware_interface.cpp [hunk 2]
```diff
@@ -533,18 +533,18 @@ void URPositionHardwareInterface::handleRobotProgramState(bool program_running)
...
   }
 
   for (size_t i = 0; i < 2; ++i) {
-    standard_analog_output_cmd_[i] = standard_analog_output_[i];
+    standard_analog_output_cmd_[i] = -1;
```
Comment (urrsk) [inline ur_robot_driver/src/hardware_interface.cpp:540]: Use constant instread of ```-1```

File: ur_robot_driver/src/hardware_interface.cpp [hunk 3]
```diff
@@ -533,18 +533,18 @@ void URPositionHardwareInterface::handleRobotProgramState(bool program_running)
...
   }
 }
 
 void URPositionHardwareInterface::checkAsyncIO()
 {
   for (size_t i = 0; i < 18; ++i) {
-    if (standard_dig_out_bits_cmd_[i] != static_cast<double>(actual_dig_out_bits_[i])) {
+    if (standard_dig_out_bits_cmd_[i] != -1) {
```
Comment (urrsk) [inline ur_robot_driver/src/hardware_interface.cpp:547]: Use constant instread of ```-1```

File: ur_robot_driver/src/hardware_interface.cpp [hunk 4]
```diff
@@ -555,16 +555,16 @@ void URPositionHardwareInterface::checkAsyncIO()
         io_async_success_ = ur_driver_->getRTDEWriter().sendToolDigitalOutput(
             static_cast<uint8_t>(i - 16), static_cast<bool>(standard_dig_out_bits_cmd_[i]));
       }
-
+      standard_dig_out_bits_cmd_[i] = -1;
```
Comment (urrsk) [inline ur_robot_driver/src/hardware_interface.cpp:558]: Use constant instread of ```-1```

File: ur_robot_driver/src/hardware_interface.cpp [hunk 5]
```diff
@@ -555,16 +555,16 @@ void URPositionHardwareInterface::checkAsyncIO()
...
       if (io_async_success_ != 2.0)
         return;
     }
   }
 
   for (size_t i = 0; i < 2; ++i) {
-    if (standard_analog_output_cmd_[i] != standard_analog_output_[i]) {
+    if (standard_analog_output_cmd_[i] != -1) {
```
Comment (urrsk) [inline ur_robot_driver/src/hardware_interface.cpp:565]: Use constant instread of ```-1```

File: ur_robot_driver/src/hardware_interface.cpp [hunk 6]
```diff
@@ -555,16 +555,16 @@ void URPositionHardwareInterface::checkAsyncIO()
...
       io_async_success_ = ur_driver_->getRTDEWriter().sendStandardAnalogOutput(i, standard_analog_output_cmd_[i]);
-
+      standard_analog_output_cmd_[i] = -1;
```
Comment (urrsk) [inline ur_robot_driver/src/hardware_interface.cpp:567]: Use constant instread of ```-1```

**Review by urrsk — CHANGES_REQUESTED** [review]
Please define -1 as a constant with a meaningful name and use that instead

File: ur_robot_driver/src/hardware_interface.cpp (hunk 1 above)
Comment (MGBla) [reply ur_robot_driver/src/hardware_interface.cpp]: Done

**Comment by MGBla** [comment]
Also fixed the same race condition as in #100 for all gpios

**Review by urrsk — APPROVED** [review]

**Pull request merged**